packages/components/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { styled } from "../../stitches.config";
+import Wrapper from "../Wrapper";
+
+export default function Footer() {
+  const nav = (
+    <Nav>
+      <a target="_blank" rel="noreferrer" href="https://app.clarity.so/%E2%82%A1ABIN">
+        Wiki
```

Comment:
could abstract this out into an `{href, label}` array and loop in case we want to add/remove nav links, as well as change up the components

Reply:
Ah, yeah. I added that additional check along with a fallback label should it not be provided.

Reply:
Made that improvement. It's now handled in the same way as in the `Navigation` component.